server/activity/hooks.js

```diff
@@ -27,6 +28,12 @@ function add(Activity, Hooks, Models) {
     sse.channel(activity.repositoryId).send(Events.Update, activity);
   }
 
+  async function sseBulkUpdate(_, { where }) {
+    const activities = await Models.Activity.findAll({ where });
+    const [activity] = activities;
+    sse.channel(activity.repositoryId).send(Events.BulkUpdate, activities);
```

Comment:
Should we make sure that all have same repositoryId (groupBy and then do multiple bulk updates; for future sake)